activerecord/lib/active_record/enum.rb

```diff
@@ -75,22 +75,22 @@ module ActiveRecord
   #
   #   Conversation.where("status <> ?", Conversation.statuses[:archived])
   #
-  # You can use the +:enum_prefix+ or +:enum_suffix+ options when you need
-  # to define multiple enums with same values. If the passed value is +true+,
-  # the methods are prefixed/suffixed with the name of the enum.
+  # You can use the +:_prefix+ or +:_suffix+ options when you need to define
+  # multiple enums with same values. If the passed value is +true+, the methods
+  # are prefixed/suffixed with the name of the enum.
   #
   #   class Invoice < ActiveRecord::Base
-  #     enum verification: [:done, :fail], enum_prefix: true
+  #     enum verification: [:done, :fail], _prefix: true
   #   end
   #
-  # It is also possible to supply a custom prefix.
+  # It is also possible to supply a custom value:
   #
   #   class Invoice < ActiveRecord::Base
-  #     enum verification: [:done, :fail], enum_prefix: :verification_status
+  #     enum verification: [:done, :fail], _prefix: :verification_status
   #   end
   #
-  # Note that <tt>:enum_prefix</tt>/<tt>:enum_suffix</tt> are reserved keywords
-  # and can not be used as an enum name.
+  # Note that <tt>:_prefix</tt>/<tt>:_suffix</tt> are reserved keywords and can
+  # not be used as enum names.
```

Comment:
We can probably :scissors: now? Seems unlikely that people would try to use `_` names, so while it's technically still true, it is just adding noise to me. What do you think?

Would trade that with an example of the feature. It doesn't really explain what the feature does, if you didn't know about the history here it seems unlikely that you would understand what it does/why you would need it from reading the docs here